Packs/JamfProtect/Integrations/JamfProtectEventCollector/JamfProtectEventCollector.py

```diff
@@ -461,18 +597,23 @@ def fetch_events(client: Client, max_fetch_alerts: int, max_fetch_audits: int, s
     audit_events, audit_next_run = [], {}
     alert_next_page = last_run.get("alert", {}).get("next_page", "")
     audit_next_page = last_run.get("audit", {}).get("next_page", "")
+    computer_next_page = last_run.get("computer", {}).get("next_page", "")
 
-    if not (alert_next_page or audit_next_page) or alert_next_page:
-        # The only case we don't trigger the alert event type cycle is when have only the audit next page token.
+    if not (alert_next_page or audit_next_page or computer_next_page) or alert_next_page:
```

Comment:
let's try to simplify the logic here (but I'm ok with leaving it as is)
- Isn't `if (x or y or z) or x` redundant? 
- we calculate `not (alert_next_page or audit_next_page or computer_next_page)` many times, can be used as a variable.